### DISCUSSION TITLE
Plotting ipf from imported orientation file

### DISCUSSION THREAD

**Discussion by amgamo1** [neperfepx/neper#381] [Q&A] [answered]
We have our own orientation file written in Euler angles as contained in the text file. We want to import this orientation to each grain of our microstructure (260 orientations for 260 grains) to plot an inverse pole figure and then visualize the mictrostructure according to the ipf color scheme. We're trying to achieve something similar to the image below. Is this possible and how should we do it?
![Inverse Pole Figure Style](https://user-images.githubusercontent.com/72865181/152563065-ea1f0de4-862b-4f96-bb43-50ee78871074.png)

Here's the orientations file we have. As we're aware, there's a specific convention the orientation file has to follow for Neper to read it but we're not sure what it should look like.
[grains.txt](https://github.com/rquey/neper/files/8004506/grains.txt)

Thank you.

**Comment by rquey** [comment, marked as answer]
First, update to version 4.3.1... Then, in -T, orientation can be loaded using [-ori](https://neper.info/doc/neper_t.html#cmdoption-ori) (updated documentation).
